coffeescript-loader/test.js

```diff
@@ -1,26 +1,32 @@
 import { ok } from 'assert';
 import { spawn } from 'child_process';
 import { execPath } from 'process';
+import { fileURLToPath, URL } from 'url';
 
 
 // Run this test yourself with debugging mode via:
 // node --inspect-brk --experimental-loader ./loader.js ./fixtures/esm-and-commonjs-imports.coffee
 
 const child = spawn(execPath, [
   '--experimental-loader',
-  './loader.js',
-  './fixtures/esm-and-commonjs-imports.coffee'
+  fileURLToPath(new URL('./loader.js', import.meta.url).href),
+  fileURLToPath(new URL('./fixtures/esm-and-commonjs-imports.coffee', import.meta.url).href),
```

Comment:
Why are these changes necessary?

Reply:
Yes, very 😉

Reply:
I don’t understand; are you saying that these changes were accidental?

Reply:
Oh, sorry you asked "why". This is a bit of an edge-case, but when the cwd is not the node-loaders project root, this fails.

For instance, in my local:

```
~/Workspace/
  …/
    nodejs/
    node-loaders/
```

I cd into the nodejs dir, which has my pr-compiled node executable, and run:

```console
./node \
--experimental-loader \
../node-loaders/coffeescript-loader/loader.js \
../node-loaders/coffeescript-loader/test.js
```

Without that extra bit of code, the files are not found.

Reply:
Okay, that’s fine then. I think the most likely real-world case would be where the loader is in `node_modules` for the current project. Like say I add `loader.js` to CoffeeScript, then you’d run something like:

```shell
node --experimental-loader ./node_modules/coffeescript/loader.js app.coffee
```

So whatever we put here should work for this case as well.

Reply:
Mm, I think the rule for this would be:

* refs to things internal to the loader module would need to be relative to the loader module (eg what you see above with `import.meta.url`)
* refs to things external to the loader module would need to be relative to the caller (`process.cwd()`).

Reply:
This breaks the tests when running on Windows with GitBash.